examples/e2ebot/data/stories.yml

```diff
@@ -1,30 +1,38 @@
 version: "2.0"
 
 stories:
-- story: happy path (intent to action)
+- story: accepts suggestion
   steps:
   - intent: greet
   - action: utter_greet
-  - intent: mood_great
-  - action: utter_happy
+  - intent: search_restaurant
+  - action: utter_suggest_cuisine
+  - intent: affirm
+  - action: utter_search_restaurants
 
-- story: sad path (text to text)
+- story: accepts 3rd suggestion
   steps:
-  - user: "[Hello](bla)"
-  - bot: "Welcome to moodbot. How are you feeling today?"
-  - user: "Horrible"
-  - bot: "Oh no! Here is a kitten photo. Did it help?"
-  - user: "Yes"
-  - bot: "Perfect"
+  - intent: search_restaurant
+  - action: utter_suggest_cuisine
+  - intent: deny
+  - action: utter_suggest_cuisine
+  - intent: deny
+  - action: utter_suggest_cuisine
+  - intent: affirm
+  - action: utter_search_restaurants
 
-- story: sad path 2 (mixed)
+- story: accepts suggestion, indirectly
   steps:
   - intent: greet
   - action: utter_greet
-  - user: "I'm not happy"
-  - action: utter_cheer_up
-  - action: utter_did_that_help
-  - user: "Not at all"
-  - bot: "Goodbye!"
-
+  - intent: search_restaurant
+  - bot: "how about sushi?"

```

Comment:
this would be a contradicting story to `accepts suggestion` and also "how about sushi?" is already part of the template, I'm afraid this example introduces more confusion on to when use e2e bot utterances

Reply:
1. why is it contradicting? it's the same story, except that the user says 'affirm' in an usual way
2. I included this because it highlights how the exact text of the bot response can also influence the meaning of the user response (e.g. if the user had said "I can always go for Italian", then they would be saying something different). 

I am totally fine with removing this, though. I think for now the main use case for full end-to-end is to train models on raw transcripts. I don't see an obvious reason for just replacing action names with bot text in your stories.

Reply:
> why is it contradicting?
you have the following steps:
```
- intent: greet
- action: utter_greet
 - intent: search_restaurant
```
in both stories and then either `utter_suggest_cuisine` or `"how about sushi?"` the bot would know which one to predict

Reply:
> I think for now the main use case for full end-to-end is to train models on raw transcripts. I don't see an obvious reason for just replacing action names with bot text in your stories.

I agree. But I thought in this example bot, we could show the raw functionality

Reply:
> I included this because it highlights how the exact text of the bot response can also influence the meaning of the user response

If I understood correctly you wanted rather to condition on bot utterance randomly picked from template response, this wouldn't work. We treat `utter_suggest_cuisine` and `"how about sushi?"` as two separate actions

Reply:
ah yes, I forgot that adding the bot utterance in this story would treat "how about sushi" as a new action rather than as a particular instance of an existing action. Something we can keep in mind for the future